packages/e2e-tests/helpers/pause-information-panel.ts

```diff
@@ -98,7 +98,18 @@ export async function openCallStackPane(page: Page): Promise<void> {
 export async function openPauseInformationPanel(page: Page): Promise<void> {
   // Only click if it's not already open; clicking again will collapse the side bar.
   const pane = getBreakpointsAccordionPane(page);
-  const isVisible = await pane.isVisible();
+
+  let isVisible = false;
+
+  try {
+    const [textContent, boundingBox] = await Promise.all([
+      pane.textContent({ timeout: 1000 }),
+      pane.boundingBox({ timeout: 1000 }),
+    ]);
+    const width = boundingBox?.width ?? 0;
+    isVisible = !!textContent && width > 0;
```

Comment:
Could we add a comment about why this unusual check is required vs just the typical `isVisible()` built-in?

Also, are both of these checks necessary or would the bounding box be sufficient?